sequel/lib/utils.js

```diff
@@ -30,6 +30,25 @@ utils.object.hasOwnProperty = function(obj, prop) {
 
 
 /**
+ * String literal helper function
+ *
+ * Pass in a value and have it wrapped in a string literal.
+ * Saves having self.escapeCharacter littered everywhere
+ *
+ * @param {String} val
+ * @param {String} escapeCharacter
+ * @return {String}
+ * @api public
+ *
+ */
+
+utils.stringLiteral = function stringLiteral(val, escapeCharacter) {
+  var regex = new RegExp(escapeCharacter, 'g');
+  return escapeCharacter+val.replace(regex, '\\'+escapeCharacter)+escapeCharacter;
```

Comment:
The concern is that malicious users can unescape the escape characters by inserting unescaped slashes.  Escaping string literals should honestly be done in reference to the connection (see https://www.npmjs.com/package/mysql#escaping-query-values), but I don't think that will be simply achieved here.  In the meantime, perhaps we should make use of `mysql.escape`?  @tjwebb ?